tests/Person.spec.ts

```diff
@@ -682,4 +683,30 @@ test.describe(`Testfälle für die Administration von Personen": Umgebung: ${pro
       await deleteRolle(page, userInfo.rolleId);
     });
   });
+
+  test("Einen Benutzer über das FE löschen @long @short @stage", async ({page, }) => {
+    const PersonManagementView = new PersonManagementViewPage(page);
```

Comment:
Notiz für später, sowas ist nicht gut. Zwei views gleichzeitig auf derselben Page haben ist semantischer Unsinn. Die Transition gehört in die Page und es wird die neue Page zurückgegeben. Aber nur als Anmerkung. In diesem Ticket lassesn wir das so.